README.md

```diff
@@ -1 +1,18 @@
-# hw01
\ No newline at end of file
+# HOMEWORK ONE   
+## Who is Paulino?
+![](https://github.com/pdiazm/hw01/blob/master/PD_Headshot.jpg)
```

Comment:
Nice use of a photo! Since the photo is in the same repo as `README.md`, you can just write this as ``![](PD_Headshot.jpg)` and it will work